mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -666,6 +636,27 @@
     <!-- Preference for site permissions -->
     <string name="preference_site_permissions">Site permissions</string>
 
+    <!--  Preference for cookie banner. The name of the item from settings screen and the title of the cookie banner screen. -->
+    <string name="preferences_cookie_banner">Cookie Banner Reduction</string>
+
+    <!--  Preference summary for the cookie banner. If the cookie banner option is not disabled. -->
+    <string name="preferences_cookie_banner_summary_on">On</string>
```

Comment:
Please briefly explain in comments what Cookie Banner is (same localizers doesn't necessarily work on same string, so this needs more context)